apps/admin/app/(auth)/forgot-password/page.tsx

```diff
@@ -0,0 +1,11 @@
+import { Metadata } from "next";
+import { ForgotPasswordForm } from "./_components/form";
+
+export const metadata: Metadata = {
```

Comment:
if the export is required for this, then shall we add it to global files?

Reply:
This is meant to be specific to each page, as the metadata needs to defined separately for each page. The minor abstraction might not be worth the effort